### PR TITLE
Unit tests audio converter

### PR DESCRIPTION
**Pull request by SaulCM**

### CODE REVIEW

File: src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package org.fundacion.jala.converter.core;
+
+import org.checkerframework.checker.units.qual.A;
+import org.fundacion.jala.converter.core.parameter.AudioParameter;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
```
Comment (JRojas999) [inline src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java]: Blank line should be deleted.

File: src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import org.checkerframework.checker.units.qual.A;
+import org.fundacion.jala.converter.core.parameter.AudioParameter;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.File;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
```
Comment (JRojas999) [inline src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java]: Specify which libraries are needed

**Review by Raymundogs — APPROVED** [review]

File: src/test/java/org/fundacion/jala/converter/core/parameter/AudioParameterTest.java [hunk 3]
```diff
@@ -0,0 +1,172 @@
+package org.fundacion.jala.converter.core.parameter;
+
+import org.fundacion.jala.converter.core.AudioConverter;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
```
Comment (JRojas999) [inline src/test/java/org/fundacion/jala/converter/core/parameter/AudioParameterTest.java]: In test, we are not enter copyright and author.

File: src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java (hunk 1 above)
Comment (SaulCM) [reply src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java]: Solved.

**Review by JRojas999 — APPROVED** [review]

File: src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java (hunk 2 above)
Comment (SaulCM) [reply src/test/java/org/fundacion/jala/converter/core/AudioConverterTest.java]: Solved.

File: src/test/java/org/fundacion/jala/converter/core/parameter/AudioParameterTest.java (hunk 3 above)
Comment (SaulCM) [reply src/test/java/org/fundacion/jala/converter/core/parameter/AudioParameterTest.java]: Solved.

**Review by JRojas999 — APPROVED** [review]

**Pull request merged**